index.html

```diff
@@ -0,0 +1,208 @@
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+            <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+            <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+            <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      </section>
+            <section class="about-container">
+              <div class="text-aline">
+                </div>
+                <div class="text-about">
+                  <div class="plane-container">
+                    <h3>About me</h3>
```

Comment:
- [ ] You have a great job in implementing the changes requested by the previous reviewer :clap: , However, you still need to pay attention on your `Margin and Padding` to match the design as this is part of the requirement.

below is the screenshot showing the difference. :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![student abt](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/43528594/13a5023d-ea91-4a2d-a933-df39df371f30)


</td>
      <td>

![Figma abt](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/43528594/bb334197-e10a-4cf9-a07d-a5ddc57780c0)


</td>
    </tr>
  </table>